EIPS/eip-7600.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 7600
+title: Hardfork Meta - Prague/Electra
+description: EIPs included in the Prague/Electra Ethereum network upgrade.
+author: Tim Beiko (@timbeiko)
+discussions-to: https://ethereum-magicians.org/t/eip-7600-hardfork-meta-prague-electra/18205
+status: Draft
+type: Meta
+created: 2024-01-18
+requires: 2537, 6110, 7002, 7549, 7569
+---
+
+## Abstract
+
+This Meta EIP lists the EIPs formally considered for and included in the Prague/Electra network upgrade. 
+
+## Specification
+
+### Included EIPs 
+
+* [EIP-6110](./eip-6110.md): Supply validator deposits on chain
+* [EIP-7002](./eip-7002.md): Execution layer triggerable exits
+* [EIP-7549](./eip-7549.md): Move committee index outside Attestation
+
+### Considered for Inclusion
+
+* [EIP-2537](./eip-2537.md): Precompile for BLS12-381 curve operations
+
+### Full Specifications 
+
+#### Consensus Layer
+
+EIPs 6110 and 7002 require changes to Ethereum's consensus layer. While the EIPs present an overview of these changes, the full specifications can be found in the `_features` directory of the `ethereum/consensus-specs` repository. 
```

Comment:
Please specify an exact link when referencing the consensus specifications.

Reply:
I think the bot complained when I did that, but let me try again...!

Reply:
@SamWilsn seems it's complaining about it.